src/app/views/ShareView.jsx

```
@@ -414,41 +430,55 @@ function BillsTable({ bills, canDispute, onRequestReview }) {
-            <div className="share-progress">
-                <div className="share-progress-bar" style={{ width: pctPaid + '%' }} />
-            </div>
-            <div className="share-progress-label">{pctPaid}% paid</div>
 
-            {ps.balanceRemaining > 0 && (
+            {/* State-driven lead (#354/#355): when settled, lead with the
+                confirmation and demote the transactional detail; when owed, lead
+                with the amount due so there's no ambiguity about what to send. */}
+            {isSettled && (
+                <div className="share-callout settled">
+                    <strong>You're all settled for {year}. Thank you!</strong>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)